src/components/ProductListContainer/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ProductCard from '../ProductCard';
+import './styles.scss';
+
+const fakeProductsList = [
+    {
+        id: 1,
+        productPhotoUrl: 'https://placehold.co/310x210?text=Imagen',
+        listPrice: '$250.000,00',
+        discountedPrice: '$200.000,00',
+        discountPercentage: 20,
+        expirationDate: new Date().toISOString(),
+        description: 'Esta es un tarjeta de producto de prueba.',
+        colors: ['azul','blanco','negro'],
+        groupCapacity: 10,
+        subscriptorsNumber: 8,
+        occupancyPercentage: 80
```

Comment:
I'm declaring these props as `Int` type 'cause I think that currently is more easy to manipulate them that way.